package.json

```diff
@@ -53,9 +53,9 @@
     "@nx/workspace": ">=16.0.0"
   },
   "dependencies": {
-    "@aws-sdk/client-s3": "3.569.0",
-    "@aws-sdk/credential-providers": "3.515.0",
-    "@aws-sdk/lib-storage": "^3.515.0",
+    "@aws-sdk/client-s3": "3.577.0",
```

Comment:
Should we also use `^` for these two AWS SDK dependencies?

Reply:
I can't find a setting in Dependabot which handles this, I think it's all driven by Dependabot.

Reply:
Oh probably it relies on the current version inside package.json. Surely, I can change this.

Reply:
Thanks for looking into this @bojanbass :)

Reply:
Thank you @terrymun!